src/common/transformations/src/transformations/common_optimizations/mark_subgraphs_to_keep_in_mixed_precision.cpp

```diff
@@ -0,0 +1,363 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/mark_subgraphs_to_keep_in_mixed_precision.hpp"
+
+#include "itt.hpp"
+#include "openvino/op/util/broadcast_base.hpp"
+#include "openvino/op/util/gather_base.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "openvino/opsets/opset2.hpp"
+#include "openvino/pass/manager.hpp"
+#include "openvino/pass/pattern/op/or.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/common_optimizations/mark_precision_sensitive_shapeof_subgraphs.hpp"
+#include "transformations/convert_precision.hpp"
+#include "transformations/rt_info/disable_fp16_compression.hpp"
+#include "transformations/utils/utils.hpp"
+
+using namespace std;
+
+namespace ov {
+namespace pass {
+
+/*
+ * MarkNormalizationOps marks MVN and NormalizeL2 to be kept in f32 precision.
+ */
+class MarkNormalizationOps : public MatcherPass {
+public:
+    OPENVINO_RTTI("MarkNormalizationOps", "0");
+
+    MarkNormalizationOps() {
+        MATCHER_SCOPE(MarkNormalizationOps);
+        auto ops_to_be_kept_fp32 = pattern::wrap_type<opset2::MVN, opset10::MVN, opset10::NormalizeL2>();
```

Comment:
can we use only opset10 here please?

Reply:
opset2::, opset10:: are not needed if you include the corresponding namespace

Reply:
Unfortunately we need need opset2 also. Normalize has only one version of the operation but MVN also has older version and we need old version of MVN to be matched with this pattern as well.

If we leave here only opset10 then we will have troubles with old IRs. Let's keep here `opset10::` for explicity

Reply:
i accepted adding using using `namespace ov::opset10` and now namespace is explicitly specified only for `opset2`